benchmarks/src/main/scala/zio/ChunkBenchmarks.scala

```diff
@@ -0,0 +1,53 @@
+package zio
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+import zio._
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+class ChunkBenchmarks {
+
+  @Param(Array("1000"))
+  var chunkSize: Int = _
+
+  @Param(Array("1000"))
+  var count: Int = _
+
+  var chunk: Chunk[Int] = _
+
+  var array: Array[Int] = _
+
+  @Setup
+  def createChunk() =
```

Comment:
One `@Setup` should suffice. Moreover, as the current layout is read-only, you should be able to initialize both of them outside of setup methods, e.g. `val array = ...`.